packages/docs/pages/reference/elements/abbr.mdx

```diff
@@ -8,12 +8,29 @@ import {
 
 # `<abbr>`
 
-Short description of what the `<abbr>` tag does.
+Wrapper for an abbreviation inside a paragraph or other block of text.  
+For example, `Many people write <abbr>Blvd.</abbr> instead of Boulevard.`
+
+Specifically for abbreviations that are neither initialisms (see [`<init>`](./init)) 
+nor acronyms (see ['<acro>`](./acro)).
```

Comment:
Here's your issue. You're typing latex style quotes! Replace `'` with a backtick.

Reply:
Got it!